test/AsmJs/uint.js

```diff
@@ -12,10 +12,9 @@ let m = function (stdlib, foreign, heap) {
   }
   return f;
 }({}, {}, asmHeap);
-print(m(4294967295));
-print(m(4294967295));
 
-m = function (stdlib, foreign, heap) {
+
+m2 = function (stdlib, foreign, heap) {
```

Comment:
I thought so. Any idea why my change caused deferred parsing on the second module though ? Can the deferred parsing order be non-deterministic and sensitive to execution times (I am guessing because the module function has no profile setup overhead, it is faster for the interpreter to get to module execution code)  ?